arm/ui/database/database.py

```diff
@@ -36,6 +36,8 @@ def view_database():
     Outputs every job from the database
      this can cause serious slow-downs with + 3/4000 entries
     """
+    global armui_cfg
```

Comment:
Not super fond of using a global for this, but it's not an issue

Reply:
I would prefer to attach it to the flask app as a config of sorts to that, but not sure how that would work. Plus would change a few pages/functions in ARM.
Unfortunately without the global I couldn't get this function to work correctly without throwing errors.